### ISSUE TITLE
[BUG] Button Message are now gone on phone Whatsapp regular app, still exist on Whatsapp web and Whatsapp Business App

### ISSUE THREAD

**Issue by isibro** [adiwajshing/Baileys#2615]
While my server still running, suddenly phone users with regular whatsapp app cannot receive **button message** , but whatsapp-web and whatsapp business users still can receive the message with button.

I know there is an update today.

Anyone know how to solve this?

UPDATE :
Today, 14 March 2023, this issue is fixed. Thanks to developer of Baileys

**Comment by 4relial** [comment]
+1

**Comment by jeferssonlemes** [comment]
For us WhatsappWeb and Iphones it's ok, but xiaomi phones are not showing the buttons

**Comment by Jakquellyn** [comment]
waiting update tomorrow

**Comment by tuin007** [comment]
same problem here

**Comment by Eximinati** [comment]
Is there any fix or update for it? Coz user can't see messages in wa messenger and wa business. Only mod wa and wa-web user can see it

**Comment by RainX-droid** [comment]
We also have this problem. Not showing in Android phone. Working fine in android an whatsapp web.

**Comment by GedeDana** [comment]
same propblem me to

**Comment by 100n0m3** [comment]
+1

**Comment by Valdenirmezadri** [comment]
+1

**Comment by zennn08** [comment]
+1

**Comment by GedeDana** [comment]
+1

**Comment by jgcvsoares2012** [comment]
Same here! Curious thing is that we didn´t find any standard scenario! Tried on 7 different cell phone. Some are fine (5 cells ), and 2 don´t receive. Tried connecting those 2 on WA Web, and messages are there! Image are receiving, single text message are fine... just Button / list Msg and specifically on few cell phones (all Android) are not working.

**Comment by Epayq** [comment]
you can downgrade whatsapp version

**Comment by jgcvsoares2012** [comment]
Should it solve it? Do you think it is related to Baileys phone WA, which is sending message with some issue? Considering that in certain phones it works fine, and other cell phones it does not (all of them were updated), I think it’s not that easy…

**Comment by brokeboienige** [comment]
+1

**Comment by batarabagus** [comment]
same issue here, it's started from a few days ago that some android phone didn't recieve button/template/list messages randomly after latest Whatsapp app update.

**Comment by enoksaju** [comment]
same issue here, working on some Android with latest app version, but on other with same version on Android does not work, not show Buttons and not Show Lists

**Comment by tuin007** [comment]
same issue here, working on some Android with latest app version, but on other with same version on Android does not work, not show Buttons and not Show Lists

**Comment by makanSukros** [comment]
Me too +1

**Comment by Bloxoid123** [comment]
+1

**Comment by ez-br** [comment]
+1

**Comment by zymaxaservices** [comment]
+1

**Comment by zymaxaservices** [comment]
same issue here, working on some Android with latest app version, but on other with same version on Android does not work, not show Buttons and not Show Lists

**Comment by morganzysman** [comment]
Any update on this ?

**Comment by marcosppastor** [comment]
+1

**Comment by isibro** [comment]
Today, this issue is fixed. Please check your app guys

**Comment by Jakquellyn** [comment]
why can't I have it?

**Comment by morganzysman** [comment]
Did you have to update anything ?

**Comment by zymaxaservices** [comment]
No need to update

**Comment by 100n0m3** [comment]
still not working here

**Comment by afeno** [comment]
Problem persist on my side as well.

**Comment by igortrust** [comment]
+1

**Comment by isibro** [comment]
> Did you have to update anything ?

no. I didn't do anything, it resolve itself..
no update, not doing any action for this issue,

By the way, I am running my server at a web hosting, not locally.

**Comment by alessandroanalistati** [comment]
for me the problem persists, buttons and list only appear on whatsapp web.

**Comment by RainX-droid** [comment]
For us. Sender does not sees the button. Only receiver sees the number.

**Comment by dyonatandias** [comment]
+

**Comment by dialogbotmx** [comment]
.

**Comment by afeno** [comment]
This issue has been closed but the problem persist. Who can re-open it?

**Comment by jgcvsoares2012** [comment]
Still same issue here!! It’s not on all cell phones! Most of them works fine. But 15% still do not receive button/list messages at all!!

**Comment by jgcvsoares2012** [comment]
Problem is not fixed - persist not working o. Some android phones…

**Comment by guiireal** [comment]
> Today, this issue is fixed. Please check your app guys

How is this resolved? Is this problem with WhatsApp? Can you describe in more detail what was fixed and how it was fixed? The more details, the better for us to be able to reproduce here. I can't even test it, because some work and others don't.

**Comment by isibro** [comment]
hi guys, I see so many user said that the problem is persist, then I re-open this.

For me, this issue is fixed, I didn't receive any comment of my client again regarding this issue..
No change on script or any other part to fix this, I was just running server as usual and then the issue fixed itself even while the server was running.

**Comment by manoeuvrelawliet** [comment]
On android phones, the probles still persists

**Comment by batarabagus** [comment]
> hi guys, I see so many user said that the problem is persist, then I re-open this.
> 
> For me, this issue is fixed, I didn't receive any comment of my client again regarding this issue.. No change on script or any other part to fix this, I was just running server as usual and then the issue fixed itself even while the server was running.

This is very random, I have two whatsapp accounts, the first account had this problem after the latest app update on android, but the second account didn't have this problem at all, then started from when you said if the problem has fixed by itself, the first one has been fixed but my second account has this problem in turn

**Comment by makanSukros** [comment]
> > hi guys, I see so many user said that the problem is persist, then I re-open this.
> > For me, this issue is fixed, I didn't receive any comment of my client again regarding this issue.. No change on script or any other part to fix this, I was just running server as usual and then the issue fixed itself even while the server was running.
> 
> This is very random, I have two whatsapp accounts, the first account had this problem after the latest app update on android, but the second account didn't have this problem at all, then started from when you said if the problem has fixed by itself, the first one has been fixed but my second account has this problem in turn

That's right, for me, when the recipient's number is not active on whatsapp web / desktop then the button appears normally on devices, but if connected to WhatsApp web / desktop then the button doesn't appear, is this related to the viewoncemessage button template? I tried the button from one of the official api, then I looked at the message template, they don't include the viewoncemessage

**Comment by allanlisboa** [comment]
+ 1

**Comment by PauloAK** [comment]
+1

**Comment by afgomezt** [comment]
+1

**Comment by alexjhr** [comment]
+1

**Comment by rafaelcw8** [comment]
+1

**Comment by rafaelcw8** [comment]
+1

**Comment by BrunoSobrino** [comment]
+1

**Comment by makanSukros** [comment]
> +1

Let's try to solve this complicated button problem together

**Comment by manoeuvrelawliet** [comment]
+1

**Comment by khusnulari1** [comment]
+1

**Comment by enoksaju** [comment]
+1

**Comment by MuhammadRestu999** [comment]
++1

**Comment by alessandroanalistati** [comment]
Aparentemente os androides mais novos não recebem os botões nem as listas.

**Comment by IanosJunior** [comment]
+1

**Comment by dxtstd** [comment]
+1

**Comment by Epayq** [comment]
I see in the group, the listbutton is replaced with a vote button and it can be seen on all devices

**Comment by khusnulari1** [comment]
> I see in the group, the listbutton is replaced with a vote button and it can be seen on all devices

Whats group bro? Can your that link

**Comment by guiireal** [comment]
> I see in the group, the listbutton is replaced with a vote button and it can be seen on all devices

This is not a list, it is a voting component... There was no official replacement, only bot creators found a way to adapt it in a "palliative" way.

**Comment by ssresh** [comment]
+!
